nemo/collections/asr/models/ctc_models.py

```diff
@@ -136,6 +145,8 @@ def transcribe(
 
         # We will store transcriptions here
         hypotheses = []
+        all_hypotheses = []
```

Comment:
Looks like it has not been used.

Reply:
Yes, for now it is unused. I need to make a breaking change in the future, such that if we have high level beam search API for CTC integrated, then the output should be a tuple (List[Hypothesis], List[NBestHypothesis]). That will require this all_hypothesis list.

However its a breaking change, and we don't support such beam search yet so I can remove this line for now and do it later. FYI @tango4j